### DISCUSSION TITLE
1EC4AFAF-022A-4461-A271-E9D083DE7D38

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#209] [General]
# C/2024 S1 (ATLAS)

C/2024 S1 (ATLAS) is a hyperbolic comet and a probable Kreutz sungrazer, discovered on 2024-Sep-27. When it reaches perihelion in 2024-Oct-28, it is possible that it will become as bright as Venus, with a magnitude between -5 and -7, as long as it can withstand the intense heat and tidal forces of the Sun.

https://celestia.mobi/addon?item=1EC4AFAF-022A-4461-A271-E9D083DE7D38&lang=zh_CN

**Comment by ASTIDA114514** [comment]
I believe that the orbit elements for this addon is outdated. It is impossible for a comet belonging to the Kreutz sungrazers to have an orbit with an eccentricity exceeding 1, and indeed, the latest orbital elements indicate an eccentricity of 0.9999145, with an orbital period reaching approximately 900 years. This requires revision(I'm from China, so I'm very bad at English XD)

**Reply by DaveBowman2001** [reply]
@SpaceExplorer2008 just posted an update a few hours ago reflecting the new orbital elements mentioned, plus ending approximately at the same time it disappeared from SOHO's coronagraph (i.e. its final disintegration)
